README.md

```diff
@@ -45,7 +45,7 @@ To contribute:
 | Hach                                                                                                                                                       | Vancouver                               | :lock: Closed :lock:  | Software Engineering – Machine Learning Co-op                                                  | Aug 27      |
 | [Ontario Teachers Pension Plan](https://otppb.wd3.myworkdayjobs.com/OntarioTeachers_Careers)                                                               | Toronto                                 | Before September 30th | Product Engineering Intern, Middleware Analyst Intern, Solution Engineering Intern - 16 Months | Aug 23      |
 | [Doordash](https://boards.greenhouse.io/doordash/jobs/5309455?gh_jid=5309455)                                                                              | Toronto                                 | Unknown               | Software Engineer, Intern - Toronto (Summer 2024)                                              | Aug 23      |
-| [Vial](https://jobs.lever.co/Vial/574e2485-a3ea-4766-861f-7adba543169a)                                                                                    | Remote                                  | Unknown               | Software Engineer Intern                                                                       | Aug 21      |
+| [Vial](https://jobs.lever.co/Vial/574e2485-a3ea-4766-861f-7adba543169a)                                                                                    | Remote                                  | :lock: Closed :lock:  | Software Engineer Intern                                                                       | Aug 21      |
```

Comment:
Should also remove the link

Reply:
Fixed! A bit late so might now be a duplication of the newer pull request made by you. But the deadline should still be an addition. Info source was UBC job board.

Reply:
great catch!